docker-compose.yaml

```diff
@@ -22,6 +22,7 @@ services:
       - './src:/home/kernelci/pipeline'
       - './config:/home/kernelci/config'
       - './data:/home/kernelci/data'
+      - './data/kcidb/:/home/kernelci/data/kcidb'
```

Comment:
The `data` directory is already mounted in the container, so this line is redundant.  It should just work with the line above.

Reply:
I have removed it